posthog/migrations/0231_add_team_person_display_name_properties.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.12 on 2022-05-02 15:16
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0230_cohort_filters"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="team",
+            name="person_display_name_properties",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(max_length=400),
+                blank=True,
+                default=["email", "name", "username"],
```

Comment:
This seems to conflict with the value in `models.py`, causing migrations to fail. I think the should revert this to `None`, and default it in the app, as migrations that touch 100+k lines (we have plenty more projects in the db than that), such as adding this default, can be rather costly.

Reply:
![image](https://user-images.githubusercontent.com/53387/166268691-e3560bbe-893c-4786-9581-f8706ef4bffd.png)
https://github.com/PostHog/posthog/runs/6260357333?check_suite_focus=true

Reply:
One test is even failing with that:

![image](https://user-images.githubusercontent.com/53387/166268784-755e661a-508d-424e-bf4b-6c3d4cf35b4a.png)

https://github.com/PostHog/posthog/runs/6260358533?check_suite_focus=true

Reply:
👍 Reverted and moved the default logic to the frontend for now